events/ri/src/main/avro/NessieEvents.avdl

```diff
@@ -0,0 +1,39 @@
+@namespace("com.example.nessie.events.generated")
+protocol NessieEvents {
```

Comment:
Does it model all the possible events or only a subset of them? 
If the latter, maybe we should document for end users what's the best way to extend it.

Reply:
It doesn't model merge and transplant events indeed, since in this example, we don't process them. I will add a note.

Reply:
Done. I added comments to all record types in the Avro schema, and also included a new record type to model Merges and Transplants (it's not used in the example, but the record definition is there). Let me know what you think.